Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+[!!LIST (Authorizations)](../UserInterface/01a_ListAuthorizations.md)
+
+
+# Monitor authorizations
+
+An authorization transaction is the permission that your company is allowed to capture money from the customer via a payment service provider. The processing of the authorization transactions works as follows: 
+-   The payment service provider creates an authorization transaction, when a customer presses the *Pay now* button during the order process.   
+- The payment provider sends then the authorization transaction to the *ActindoCore1 Platform*.    
+- In addition, the shop system creates an ID for this transaction.  
+- Actindo links both.   
+
+For Actindo, the authorization transaction serves as a reference to a payment made by a customer.   
+You can use this procedure to display all incoming authorization transactions of a payment service provider. You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example. In addition, if applicable, you can change the status of a transaction to start continuing further processing or to end it.    
+
```

Comment:
You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example. In addition, if applicable, you can change the status of a transaction to start continuing further processing or to end it. 

Ich bin mir nicht sicher, ob das so stimmt.